src/reporting/Market.sol

```diff
@@ -416,6 +416,12 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
             return ReportingState.FINALIZED;
         }
 
+        // If there is an active fork and we're past PRE-REPORTING we need to migrate
```

Comment:
Even `PRE_REPORTING` markets need to be migrated.  It doesn't appear that we currently use `PRE_REPORTING` (though I think it is a valuable state), so I think for clarity we should change this comment to just be,
> If there is an active fork and we aren't finalized, we need to migrate.

Whether we label the market as `PRE_REPORTING` or `AWAITING_MIGRATION` doesn't really matter to me, but technically the market is `PRE_REPORTING & AWAITING_MIGRATION`.

Reply:
We'll need to move this up actually if that's the case. See above: 

> if (block.timestamp < endTime) {
>     return ReportingState.PRE_REPORTING;
>  }

Reply:
We can return _either_ `PRE_REPORTING` or `AWAITING_MIGRATION` for markets in this state and the system will be correct.  The `PRE_REPORTING` check is cheaper, so my preference is to leave it up there and this one down here.  My concern is just that the comment tells the reader that being past `PRE_REPORTING` is a requirement.